File: examples/docs-code-snippets/src/main/java/examples/java/recipes/RecipeWithRetentionPeriod.java
package examples.java.recipes;

import com.ing.baker.recipe.javadsl.Recipe;

import java.time.Duration;

public class RecipeWithRetentionPeriod {

    public final static Recipe recipe = new Recipe("example")
        .withRetentionPeriod(Duration.ofDays(3));
}

File: examples/docs-code-snippets/src/main/kotlin/examples/kotlin/recipes/RecipeWithRetentionPeriod.kt
package examples.kotlin.recipes

import com.ing.baker.recipe.kotlindsl.ExperimentalDsl
import com.ing.baker.recipe.kotlindsl.recipe
import kotlin.time.Duration.Companion.days
import kotlin.time.Duration.Companion.hours

@ExperimentalDsl
object RecipeWithRetentionPeriod {
    val recipe = recipe(name = "example") {
        retentionPeriod = 3.days
    }
}